README.md

```diff
@@ -127,6 +127,12 @@ To use a pre-existing EC2 key pair in `us-east-1` region, you can specify key pa
 eksctl create cluster --ssh-public-key=my_kubernetes_key --region=us-east-1
 ```
 
+To add custom tags to each stack, you can specify tags: 
```

Comment:
I'd say "To add custom tags for all resources, use `--tags`:" instead, because stacks aren't really discussed in the readme (unless you want to extend it a bit more)...